src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,211 @@
+
+{% Aside %}
+Why are we calling this a "reverse" origin trial? Normally, an origin trial allows developers to opt-in to early access to new functionality before it has shipped by default in Chrome. In this case, we're allowing developers to opt-in to using legacy technology even after it's been removed from Chrome.
+{% endAside %}
+
+To participate in the "reverse" origin trial:
+
+- [Request a token](https://developers.chrome.com/origintrials/#/register_trial/1776670052997660673) for your origin.
+
+- Add the token to your HTML pages. There are [two ways](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#how-do-i-enable-an-experimental-feature-on-my-origin) to do that:
+
+  - Add an `origin-trial` `<meta>` tag to the head of each page. For example: `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+
+  - Alternatively, configure your server to return responses containing the `Origin-Trial` HTTP header. The resulting response header should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+- Add the same token to your AppCache manifests. This is done via a new field in your manifest, with the format:
+
+```text
+ORIGIN-TRIAL:
+TOKEN_GOES_HERE
+```
+
+(There needs to be a new line between `ORIGIN-TRIAL:` and your token.)
+
+{% Aside %}
+The token for a manifest **must** be in an `ORIGIN-TRIAL:` field of the manifest itself. Unlike an HTML page's token, it can't be provided via an HTTP header.
+{% endAside %}
+
+You can see a sample project embedded below which demonstrates adding the correct origin trial tokens into both the `index.html` and `manifest.appcache` files.
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe src="https://glitch.com/embed/#!/embed/appcache-reverse-ot?path=manfiest.appcache&attributionHidden=true"
+          alt="An example web app that is registered for the AppCache origin trial."
+          style="height: 100%; width: 100%; border: 0;"></iframe>
+</div>
+
+### Why are tokens needed in multiple places?
+
+The **same origin trial token** needs to be associated with:
+
+- **All of your HTML pages** that use AppCache.
+- **All of your AppCache manifests** via the `ORIGIN-TRIAL` manifest field. 
+
+If you've participated in origin trials in the past, you might have added the token just to your HTML pages. The AppCache "reverse" origin trial is special in that you need to associate a token with each of your AppCache manifests as well.
```

Comment:
Is this particular to the AppCache reverse origin trial or reverse origin trials in general. If the latter, please remove 'AppCache' from this sentence.

Reply:
It's the former, so I'm going to leave "AppCache" in the sentence.